addons/test_website/static/tests/field_html_file_upload.js

```diff
@@ -95,13 +97,34 @@ QUnit.module('field html file upload', {
                 </form>`,
         };
         const mockRPC = (route, args) => {
-            if (route === "/web_editor/attachment/add_data") {
-                return Promise.resolve({"id": 5, "name": "test.jpg", "description": false, "mimetype": "image/jpeg", "checksum": "7951a43bbfb08fd742224ada280913d1897b89ab",
-                                        "url": false, "type": "binary", "res_id": 1, "res_model": "note.note", "public": false, "access_token": false,
-                                        "image_src": "/web/image/1-a0e63e61/test.jpg", "image_width": 1, "image_height": 1, "original_id": false
-                                        });
-            }
-            else if (route === "/web/dataset/call_kw/ir.attachment/generate_access_token") {
+        if (args.method === "web_save") {
```

Comment:
Could you move this in `web_editor`?
This has nothing to do with `website` module, it's a fix in `web_editor` for the composer.

It also really not have anything to do with the `test_website` module.
`test_website` purpose is to be able to test behavior in test which would require real elements to be added in the DB which can't be done in a regular module, like some routes, some views, some models etc which we need to be encapsulated in a separate module and there is no workaround it.

Unless your fix has anything to do with `web_unsplash`?
Initial test here was introduced because it needed a bridge for web_editor and web_unsplash, which was okay~ish but if we introduce some fix for web_editor, let's add the test in.. web_editor please.

Reply:
@rdeodoo indeed web_unsplash is why this test was added in that module to start with.

The issue is that if web_unsplash is installed at all, then I need to initialize the service so the image picker can actually open. But if it's not installed, then trying to start it will fail.

I think it's good best if tests work consistently with only the module they're in and its dependencies.

If you have some idea to conditionally import it I could go with that. Currently runbot will crash if I put the test in the "correct" file.